SeleniumCookies/wrapper.py

```diff
@@ -329,7 +362,8 @@ def __add_session_cookies_lz4(self, cj):
             file_obj.read(8)
             json_data = json.loads(lz4.block.decompress(file_obj.read()))
         except ValueError as e:
-            logging.error(f'Error parsing firefox session JSON LZ4: str(e)')
+            logging.error("{'ErrorCode':'%s' ,'Error':'%s'}",
+                          LZ4ParsingError, CodeMap[LZ4ParsingError])

```

Comment:
Just pass the code and let the wrapper deal with it on its own. The same wrapper which judges if logs should be silent in production or not, according to the level

Reply:
to do this i will have to be creating a function, so now a function with code will have to passed inside the logging message. Do you want it like to be this way? @L04DB4L4NC3R

Reply:
@D-E-F-E-A-T new information has been uncovered. Apparently debug is automatically silenced in production mode. Just configure the logger according to the `ENV`. No need to create new funcitons

Reply:
According to the `ENV`, use `logging.addLevelName` to add only the required attributes

Reply:
How can it be known that it's in a production environment? It can be changed from logging.level though
@L04DB4L4NC3R

Reply:
You can define env variable to check prod env or dev env. Check using os module and configure logger accordingly.

Reply:
@Geek-ubaid thanks. woking on it,

Reply:
@L04DB4L4NC3R @Geek-ubaid Someone fix this